target/ppc/insn32.decode

```diff
@@ -711,6 +726,23 @@ XVTLSBB         111100 ... -- 00010 ..... 111011011 . - @XX2_bf_xb
 @XL_s           ......-------------- s:1 .......... -   &XL_s
 RFEBB           010011-------------- .   0010010010 -   @XL_s
 
+## SLB Management Instructions
+
+SLBIE           011111 ----- ----- ..... 0110110010 -   @X_rb
+SLBIEG          011111 ..... ----- ..... 0111010010 -   @X_sb
+
+SLBIA           011111 --... ----- ----- 0111110010 -   @X_ih
+SLBIAG          011111 ..... ----. ----- 1101010010 -   @X_rs_l
+
+SLBMTE          011111 ..... ----- ..... 0110010010 -   @X_sb
+
+SLBMFEV	        011111 ..... ----- ..... 1101010011 -   @X_sb
+SLBMFEE	        011111 ..... ----- ..... 1110010011 -   @X_sb
```

Comment:
The L bit is still missing in the decode of these 2 instructions, but since it is implementation specific and not used while translating the instruction, I guess it's ok to leave it as a dash.